src/manager.js

```diff
@@ -24,17 +26,18 @@ var AdManager = function(options) {
   this.adId = 0;
   this.initialized = false;
   this.viewportWidth = 0;
-  this.oldViewportWidth = window.document.body.clientWidth;
+  this.oldViewportWidth = window.innerWidth;
```

Comment:
What was the impetus behind making this change?  Those values are different, and `innerWidth` includes the width of the scroll bar, which I wouldn't think we'd want to include.